backend/database/versions/07_e31a29561f38_add_is_migration_revision_flag.py

```diff
@@ -0,0 +1,27 @@
+"""add is_migration_revision flag
+
+Revision ID: 07_e31a29561f38
+Revises: 06_8bced1b1470b
+Create Date: 2024-04-23 10:32:09.778855
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "07_e31a29561f38"
+down_revision = "06_8bced1b1470b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "CollectionVersion",
+        sa.Column("is_migration_revision", sa.BOOLEAN(), nullable=True),
```

Comment:
note from Emanuele: rename to something that does not expose business logic terms ("migration" and "revision") to DB. Suggested: `is_auto_version`